experimental/LieAlgebras/src/Combinatorics.jl

```diff
@@ -3,11 +3,11 @@
 # more efficient implementation.
 
 function combinations(n::Integer, k::Integer)
-  return sort(subsets(n, k))
+  return sort(AbstractAlgebra.combinations(n, k))
```

Comment:
That's an unrelated change? (Not that I terribly mind, just am surprised)

Reply:
When fixing the calls to this method outside of Oscar.LieAlgebras I found this as an alternative. It does not make any speed difference in the limited places it is called anyway. So this change is more a note to me to see if I can get rid of Oscar.LieAlgebras.combinations in the near future.